clang/lib/Format/UnwrappedLineParser.cpp

```diff
@@ -370,9 +364,9 @@ bool UnwrappedLineParser::parseLevel(const FormatToken *OpeningBrace,
     else if (FormatTok->getType() == TT_MacroBlockEnd)
       kind = tok::r_brace;
 
-    auto ParseDefault = [this, OpeningBrace, NextLevelLBracesType, IfKind,
+    auto ParseDefault = [this, OpeningBrace, InRequiresExpression, IfKind,
                          &IfLBrace, &HasDoWhile, &HasLabel, &StatementCount] {
-      parseStructuralElement(!OpeningBrace, NextLevelLBracesType, IfKind,
+      parseStructuralElement(OpeningBrace, InRequiresExpression, IfKind,
```

Comment:
If `InRequiresExpression` can be determined from just `OpeningBrace`, does it need to be passed down here?

Reply:
Nope!